config/ignition_fortress_ubuntu_focal.yaml

```diff
@@ -41,14 +38,6 @@ filter_formula: "\
  Package (= libignition-launch5) |\
  Package (= libignition-launch5-dev) \
 ), $Version (% 5.1.0-1*) |\
-(Package (= ignition-math6) |\
- Package (= libignition-math6) |\
- Package (= libignition-math6-dbg) |\
- Package (= libignition-math6-dev) |\
- Package (= libignition-math6-eigen3-dev) |\
- Package (= python3-ignition-math6) |\
- Package (= ruby-ignition-math6) \
-), $Version (% 6.10.0-1*) |\
```

Comment:
I would prefer that these changes are revered for the reasons described in https://github.com/ros-infrastructure/reprepro-updater/pull/168#issuecomment-1218663623

Reply:
Reverted [adf0dce](https://github.com/ros-infrastructure/reprepro-updater/pull/168/commits/adf0dcee76545baaff671047521d91dad91e8176)